torch/csrc/jit/attributes.h

```diff
@@ -19,10 +19,21 @@ using ::c10::Symbol;
 
 constexpr int max_tensor_display_size = 10;
 
-enum class AttributeKind { f, fs, i, is, s, ss, t, ts, g, gs, ty, tys };
+enum class AttributeKind { f, fs, i, is, s, ss, t, ts, g, gs, ty, tys, ival };
```

Comment:
nit: can you name it `ivalue` instead of `ival`? The abbreviations here are already pretty messy and unclear (like what is `tys`?)

Reply:
`ty` is type, and `tys` is list of `ty`.